src/erldns_storage_json.erl

```diff
@@ -49,6 +49,8 @@ create(Name = host_throttle) ->
 create(Name = lookup_table) ->
   create_ets_table(Name, bag);
 create(Name = handler_registry) ->
+  create_ets_table(Name, set);
+create(Name = sync_counters) ->
```

Comment:
You will also need to implement this in `erldns_storage_mnesia`.

Reply:
commit: https://github.com/dnsimple/erldns/commit/4a9219b8b55c2a50dd74bbb7dc264f306497ceec

covers mnesia storage backend.